site/docs/v1/tech/guides/passwordless.adoc

```diff
@@ -37,9 +47,9 @@ image::guides/passwordless/turn-on-passwordless.png[Turn on passwordless login,w
 
 Since, by default, passwordless authentication requires email delivery, ensure FusionAuth is correctly configured to send email. One way to test email delivery is by creating a user and sending them a password setup email.
 
-== Using the FusionAuth OAuth interface
+== Using the FusionAuth OAuth Interface
 
-Choose the FusionAuth OAuth interface approach if you want to use the link:/docs/v1/tech/oauth/#example-authorization-code-grant[Authorization Code grant]. Enabling passwordless adds an option for a user to receive a one-time code. Because this is the Authorization Code grant, any library or framework that supports this OAuth grant will work.
+Choose the FusionAuth OAuth interface, or hosted login pages, approach if you want to use the link:/docs/v1/tech/oauth/#example-authorization-code-grant[Authorization Code grant]. Enabling passwordless adds an option for a user to receive a one-time code. Because this is the Authorization Code grant, any library or framework that supports this OAuth grant will work.
```

Comment:
Does adding commas for "or hosted login pages" make this read funny?  I feel like we should omit them.
```suggestion
Choose the FusionAuth OAuth interface or hosted login pages approach if you want to use the link:/docs/v1/tech/oauth/#example-authorization-code-grant[Authorization Code grant]. Enabling passwordless adds an option for a user to receive a one-time code. Because this is the Authorization Code grant, any library or framework that supports this OAuth grant will work.
```